website/content/docs/ecs/manual/secure-configuration.mdx

```diff
@@ -52,22 +47,80 @@ service_prefix "" {
 }
 ```
 
-This policy allows `node:write` for any node name, which is necessary because the Consul node
-names on ECS are not known until runtime.
+The policy allows `node:write` for any node name, which is necessary because the Consul node names on ECS are not known until runtime.
+
+You can add the policy in Consul using the [`consul acl policy create`](/commands/acl/policy/create) command or the [`[PUT] /v1/acl/policy`](/api-docs/acl/policies#create-a-policy) API endpoint.
+
+If you intend to create a gateway for connecting multiple Consul datacenters, you will need additional policies to specify the permission scope.
+
+#### Mesh gateway policy
+
+Mesh gateways must run in the default namespace.
+
+```hcl
+namespace "default" {    ## If namespaces enabled
+    service "<Service Name>" {
+       policy = "write"
+    }
+}
+namespace_prefix "" {    ## If namespaces enabled
+    service_prefix "" {
+       policy = "read"
+    }
+    node_prefix "" {
+       policy = "read"
+    }
+}
+agent_prefix "" {
+   policy = "read"
+}
+```
+#### Terminating gateway policy
```

Comment:
We don't yet support running terminating and ingress gateways on ECS, so we shouldn't include the policies for those gateways here.